docs/reference/contrib/redirects.md

```diff
@@ -10,7 +10,8 @@ The `redirects` module provides the models and user interface for managing arbit
 
 ## Installation
 
-The `redirects` module is not enabled by default. To install it, add `wagtail.contrib.redirects` to `INSTALLED_APPS` and `wagtail.contrib.redirects.middleware.RedirectMiddleware` to `MIDDLEWARE` in your project's Django settings file.
+The `redirects` module is enabled by default when you create a new Wagtail website using the `wagtail start` command. However, if you are upgrading from an older version of Wagtail you may need to manually install it. To install it, add `wagtail.contrib.redirects` to `INSTALLED_APPS` and `wagtail.contrib.redirects.middleware.RedirectMiddleware` to `MIDDLEWARE` in your project's Django settings file.
```

Comment:
```suggestion
To install this module, add `wagtail.contrib.redirects` to `INSTALLED_APPS` and `wagtail.contrib.redirects.middleware.RedirectMiddleware` to `MIDDLEWARE` in your project's Django settings file.
```

Reply:
@KalobTaulien I'm sure you appreciate the quandary, though? It ultimately comes down to consistency... If we go explaining the difference here, what about the other apps included by the `wagtail start` command? And what if we change our mind about what the start command does and doesn't include?

It seems sensible to me to make as few assumptions as possible in topic-specific documentation. I think it should be down to the start command documentation to outline what it does and doesn't include.